<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>splash</class>
 <widget class="QMainWindow" name="splash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>splash</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1171</width>
      <height>811</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/pictures/2.jpg)
size(300,300)</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="firstTab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>700</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #3498db;       /* آبی جذاب */
    color: white;                    /* متن سفید */
    border: none;
    border-radius: 10px;             /* گوشه‌های گرد */
    padding: 8px 20px;               /* فضای داخلی */
    font-size: 14px;
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #2980b9;       /* آبی تیره‌تر هنگام هاور */
}

QPushButton:pressed {
    background-color: #2471a3;       /* هنگام کلیک */
    padding-left: 22px;              /* جلوه کلیک‌دار */
    padding-top: 9px;
}

QPushButton:disabled {
    background-color: #bdc3c7;
    color: #7f8c8d;
}
</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="StartScreenTab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="GalleryButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>590</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Gallery Agents</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MainPageButton">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>590</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>PLAY!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="GalleryTab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="BackButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic23">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>91</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>293</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Sans Mono</family>
        <pointsize>22</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Billy.webp)
size(64,64)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pic24">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>90</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Sanka.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pic2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Reketon.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pic3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>90</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Angus.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pic4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Duraham.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic6">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>90</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Medusa.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic1">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>90</y>
        <width>131</width>
        <height>102</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>4</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Colonel_baba.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pic8">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>90</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Bunka.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pic5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Sir philiph.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Kabu.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic9">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Rajakal.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic10">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>240</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Salih.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pic11">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>240</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Khan.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic12">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>240</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Eloi (1).webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic13">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>240</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Boi.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic14">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>640</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/death.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>640</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Sabrina.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Rambo.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Elsa.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic18">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>240</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Kanar.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic19">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>350</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Sir philiph.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic20">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>350</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Frost.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic21">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>350</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Tusk.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic22">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Agents/Karissa.webp)</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>599</y>
        <width>1161</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background:rgb(119, 118, 123);
font: 700 11pt &quot;OpenSymbol&quot;;</string>
         </property>
         <property name="text">
          <string>Flying</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>1161</width>
        <height>45</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">background:rgb(99, 69, 44);
font: 700 italic 11pt &quot;Noto Serif Tibetan&quot;;</string>
         </property>
         <property name="text">
          <string>Grounded</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>1161</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">background:rgb(53, 132, 228);
font: 800 11pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Water Walking</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>740</y>
        <width>1161</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">background:rgb(192, 97, 203);
font: 800 italic 11pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Floating</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="mainPlayGameTab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>First Player</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>140</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Second Player</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>360</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
